frontends/election-manager/src/utils/cast_vote_record_files.ts

```diff
@@ -185,19 +185,20 @@ export class CastVoteRecordFiles {
   /**
    * Parses the filenames for the given CVR {@link files} for information about
    * the CVRs they contain.
+   *
+   * TODO: After a few rounds of refactoring, this is now mostly just a proxy to
+   * {@link parseCvrFileInfoFromFilename} - should consolidate at some point.
```

Comment:
Is the rest of this module still used now that the data is loaded from the backend?

Reply:
okey doke